products/cakes/cakeA.html

```diff
@@ -0,0 +1,297 @@
+<!DOCTYPE HTML>
```

Comment:
let's put the `cakes` folder in the `menu` folder that currently exist on the main branch. Same for the future `cookies`, `cupcakes`, and `pies` folders and their respective products.

Reply:
sounds good, thanks! confirming I fixed the paths with the following commit.